cellxgene_resources/curation_qa.ipynb

```diff
@@ -455,20 +446,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "prop = 'sample_id'\n",
-    "\n",
-    "results = []\n",
-    "for a in adata.obs[prop].unique():\n",
-    "    print(a)\n",
-    "    obs_df = adata.obs[adata.obs[prop] == a]\n",
-    "    num_to_check = obs_df.shape[0] // 5 #default is check 20% of barcodes\n",
-    "    r = TENx_barcode_checker(obs_df, num_to_check)\n",
-    "    if r:\n",
-    "        r[prop] = a\n",
-    "        results.append(r)\n",
-    "    else:\n",
-    "        print('no barcodes checked')\n",
-    "pd.DataFrame(results).set_index(prop)"
+    "num_to_check = adata.obs.shape[0] // 5 #default is check 20% of barcodes\n",
+    "csv = '10X_barcode_table.csv'\n",
```

Comment:
we don't have a use case that would require any other csv be used, so I would remove this as a variable/function parameter and just put it directly into the function

Reply:
Should probably also go ahead and generate the csv & include that in the PR.
The code to generate it will be convenient in the sample_code ntbk, but we can still prevent everyone from having to run it.

Reply:
i tried to push a zipped version of the csv, but it was too large: 
`remote:error: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.`

Reply:
nevermind, the gzip version worked, the non-zipped file was the issue